public/apps/configuration/panels/get-started.tsx

```diff
@@ -0,0 +1,171 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import {
+  EuiButton,
+  EuiPageHeader,
+  EuiText,
+  EuiTitle,
+  EuiSteps,
+  EuiCode,
+  EuiSpacer,
+  EuiImage,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiLink,
+} from '@elastic/eui';
+import React, { Fragment } from 'react';
+import { AppDependencies } from '../../types';
+import securitysteps from '../../../assets/get_started.png';
+import { buildHashUrl } from '../utils/url-builder';
+import { Action, ResourceType } from '../types';
+
+const setOfSteps = [
+  {
+    title: 'Secure your backends (authc & authz)',
+    children: (
+      <Fragment>
```

Comment:
Suggest be consistently using shorthand `<>` instead of `<Fragment>`. Same comments for below.

Reply:
Will use `<>` instead of `<Fragment>` to be consistent.